client/src/App.jsx

```diff
@@ -23,7 +25,7 @@ const App = ({ history, location }) => (
 );
 
 App.propTypes = {
-  history: PropTypes.shape().isRequired,
-  location: PropTypes.shape().isRequired,
+  // history: PropTypes.shape().isRequired,
+  // location: PropTypes.shape().isRequired,
```

Comment:
Why commenting these? This isn't related to the Navbar.

Reply:
Throwing an error. History and location are undefined. I guess Hasan has to still work on those.